release/pkg/helm/helm.go

```diff
@@ -454,7 +448,9 @@ func OverWriteChartValuesImageSha(filename string, shaMap map[string]anywherev1a
 		return err
 	}
 	values["controller"].(map[string]interface{})["digest"] = shaMap["eks-anywhere-packages"].ImageDigest
+	values["controller"].(map[string]interface{})["tag"] = shaMap["eks-anywhere-packages"].ImageDigest
```

Comment:
before modifying this helm chart, can we check if the varlues are even different and if they are not, dont do this at all?

also, if we do push a new helm chart, i think the sha in the bundle should reflect this modified and pushed chart vs what it is now, the original chart.

Reply:
I did one check, versions in bundle-release.yaml and chart's values.yaml are matching, but we cannot induce that all bundle-release.yaml are good. There might be some good reason why this modification exists.

Right, we need to use the new digest in dev-bundle-release.yaml's packages. I can do it in a separate PR.

Reply:
In fact, let's try removing the modification

Reply:
No way this is for all charts? I don't really follow, all I'm saying is if the digest it's about to set is what's already in the values yaml for the chart sr pushed there is no reason to update

We do need to check with Jonah and try to understand what issue was being solved by making them tags instead of digests, but if we want to support airgapped I think ultimately we need to use digests here. 

I also wonder if we should change the cli to pull the helm chart being sha instead of tag to ensure we are installing the same one referenced in the bundle.